docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1401,48 +1409,12 @@ public interface EasyScoreCalculator<Solution_, Score_ extends Score<Score_>> {
 }
 ----
 
-For example in N-queens:
-
-[source,java,options="nowrap"]
-----
-public class NQueensEasyScoreCalculator
```

Comment:
Let's add an EasyScoreCalculator code snippet.

Reply:
Decided not to. People should be writing constraint providers; easy is still there, but let's not draw attention to it.